.clang-tidy

```diff
@@ -0,0 +1,306 @@
+---
+# -cppcoreguidelines-pro-type-vararg: It will alert on every ASSERT.
```

Comment:
Can you add some more comments about how this file was generated / where it comes from?

Reply:
Done